docs/user-guide/release-notes/3.x.x.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Pulp 3 Release Notes
+=======================
+
+Pulp 3
+======
+
+Changelog
+---------
+
+* Nodes are no longer supported. And were completly removed please use just sync from one pulp
+  server to another instead of this.
```

Comment:
Thanks for adding these. I couldn't help but elaborate on the language here. You can include it if you want.

```
 Nodes are no longer supported and were completely removed with Pulp 3.0. Instead of using nodes, publish content from one Pulp server and use that feed url to sync the content into another Pulp server. This is the recommended approach for a nodes use case.
```